root/multicore/loadImt.cpp

```diff
@@ -1,6 +1,8 @@
 #include "TSystem.h"
 #include "TInterpreter.h"
+#ifndef _MSC_VER
```

Comment:
Did we used to use ```_WIN32``` (and ```R__WIN32```) for this purpose?

Reply:
Well, I prefer `_MSC_VER` it's more explicit for `Visual Studio` cases (there is also `clang` and `MingW gcc`)

Reply:
Fair enough ....